backend/src/schema/resolvers/rewards.js

```diff
@@ -1,47 +1,47 @@
-export default {
-  Mutation: {
-    reward: async (_object, params, context, _resolveInfo) => {
-      const { fromBadgeId, toUserId } = params
-      const session = context.driver.session()
-
-      let transactionRes = await session.run(
-        `MATCH (badge:Badge {id: $badgeId}), (rewardedUser:User {id: $rewardedUserId})
-          MERGE (badge)-[:REWARDED]->(rewardedUser)
-          RETURN rewardedUser {.id}`,
-        {
-          badgeId: fromBadgeId,
-          rewardedUserId: toUserId,
-        },
-      )
+import { neode } from '../../bootstrap/neo4j'
+import { UserInputError } from 'apollo-server'
 
-      const [rewardedUser] = transactionRes.records.map(record => {
-        return record.get('rewardedUser')
-      })
+const instance = neode()
 
-      session.close()
+const getUserAndBadge = async ({ badgeKey, userId }) => {
+  let user = await instance.first('User', 'id', userId)
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-12T07:34:13Z" title="Friday, July 12th 2019, 9:34:13 am +02:00">Jul 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,47 +1,47 @@
-export default {
-  Mutation: {
-    reward: async (_object, params, context, _resolveInfo) => {
-      const { fromBadgeId, toUserId } = params
-      const session = context.driver.session()
-
-      let transactionRes = await session.run(
-        `MATCH (badge:Badge {id: $badgeId}), (rewardedUser:User {id: $rewardedUserId})
-          MERGE (badge)-[:REWARDED]->(rewardedUser)
-          RETURN rewardedUser {.id}`,
-        {
-          badgeId: fromBadgeId,
-          rewardedUserId: toUserId,
-        },
-      )
+import { neode } from '../../bootstrap/neo4j'
+import { UserInputError } from 'apollo-server'
 
-      const [rewardedUser] = transactionRes.records.map(record => {
-        return record.get('rewardedUser')
-      })
+const instance = neode()
 
-      session.close()
+const getUserAndBadge = async ({ badgeKey, userId }) => {
+  let user = await instance.first('User', 'id', userId)
```


`const` will make it?